apps/fabric-website/scripts/createFlightConfig.js

```diff
@@ -0,0 +1,63 @@
+/*
+  Creates a configuration JS file that exposes the build ID and base URL that the site should load.
+ */
+
+const fs = require('fs');
+const path = require('path');
+const { argv, logger } = require('@uifabric/build/just-task').just;
+
+module.exports = function createFlightConfigTask() {
+  return function createFlightConfigTask() {
```

Comment:
shadowed name?

Reply:
oi... i guess - i can get rid of the name of the first one.